src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -191,6 +191,9 @@ impl Coordinator {
                         // Subsources use source statuses.
                         DataSourceDesc::Source => (DataSource::Other, source_status_collection_id),
                         DataSourceDesc::Progress => (DataSource::Progress, None),
+                        DataSourceDesc::Webhook => {
+                            unreachable!("Webhook sources take different path")
```

Comment:
Can you add a TODO or a link here to the GitHub epic for webhooks? Eventually we will definitively want webhooks to take this path, e.g. once they're running in dataflows.

Reply:
Added!